test/test-get-one-week-arrival.js

```diff
@@ -0,0 +1,63 @@
+const assert = require('assert');
+
+const request = require('supertest');
+const app = require('../server');
+
+const mongodbMemoryServer = require('./util/mongodb-in-memory');
+const mockData = require('./util/mock-data');
+const mongoClientStub = {};
+const proxyRequire = require('proxyquire');
```

Comment:
We can remove this lines since were no longer stubbing the method that connects the mongodb in memory.